EKS_DISTRO_TAG_FILE.yaml

```diff
@@ -14,17 +14,17 @@ al2:
   eks-distro-minimal-base-nsenter: 2022-07-27-1658910674.2
   eks-distro-minimal-base-python3.9: 3.9-2022-10-11-1665514870.2
 al2022:
-  eks-distro-base: 2022-10-13-1665687739.2022
-  eks-distro-minimal-base: 2022-10-13-1665687739.2022
-  eks-distro-minimal-base-nonroot: 2022-10-13-1665687739.2022
-  eks-distro-minimal-base-glibc: 2022-10-13-1665687739.2022
-  eks-distro-minimal-base-iptables: 2022-10-13-1665687739.2022
-  eks-distro-minimal-base-docker-client: 2022-10-13-1665687739.2022
-  eks-distro-minimal-base-csi: 2022-10-13-1665687739.2022
-  eks-distro-minimal-base-csi-ebs: 2022-10-13-1665687739.2022
-  eks-distro-minimal-base-haproxy: 2022-10-13-1665687739.2022
-  eks-distro-minimal-base-kind: 2022-10-13-1665687739.2022
-  eks-distro-minimal-base-nginx: 2022-10-13-1665687739.2022
-  eks-distro-minimal-base-git: 2022-10-13-1665687739.2022
-  eks-distro-minimal-base-nsenter: 2022-10-13-1665687739.2022
-  eks-distro-minimal-base-python3.9: 3.9-2022-10-13-1665687739.2022
+  eks-distro-base: null
```

Comment:
Are these null now because the install below now handles it?

Reply:
null so that the next peridoic job will pick them up and rebuild them all